ioctl/newcmd/account/accountdelete.go

```diff
@@ -102,17 +100,15 @@ func NewAccountDelete(c ioctl.Client) *cobra.Command {
 			ks := c.NewKeyStore(config.ReadConfig.Wallet, keystore.StandardScryptN, keystore.StandardScryptP)
 			for _, v := range ks.Accounts() {
 				if bytes.Equal(account.Bytes(), v.Address.Bytes()) {
-					var confirm string
 					message := output.ConfirmationMessage{Info: infoWarn, Options: []string{"yes"}}
 					fmt.Println(message.String())
-					fmt.Scanf("%s", &confirm)
-					if !strings.EqualFold(confirm, "yes") {
+					if !c.AskToConfirm() {
```

Comment:
if the command line has `-y`, that means user confirms ahead. In that case, don't need to call `AskToConfirm`, pls take that into consideration, things like:
1. can this check be done inside `AskToConfirm`? if yes, that's the best
2. if not, what are the solution? like add a `needConfirm` flag to command, and call it for those commands with the flag? or other alternatives?

Reply:
i can add a new flag to check wether user write `-y`, and check this flag before `AskToConfirm`.  if has, delete account directly;  if not, ask user to confirm.  
in current version,  `-y` is not supported, i think this may be safer for deleting account cannot be recovered and avoid user write wrong account by accident. 
so can we need add `-y` ?

Reply:
not add, keep the same with ioctl/cmd